samples/bluetooth/connection_event_trigger/README.rst

```diff
@@ -0,0 +1,165 @@
+.. _ble_connection_event_trigger:
+
+Bluetooth: Event Trigger
+########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Bluetooth® Event Trigger sample showcases the Event Trigger feature, a proprietary
+extension of the Bluetooth® protocol which can be used to schedule sampling, GPIO toggling, or other SW- or HW-based
+user activities in synchronization with individual BLE role instances, such as a central or peripheral link.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-sample-yaml::
+
+You can use any two of the development kits mentioned above and mix different development kits.
+
+Additionally, the sample requires a connection to a computer with a serial terminal for each of the development kits.
+
+Note that the Event Trigger feature involves triggering a (D)PPI task directly from the SoftDevice Controller link layer,
+and therefore it is expected that the application and SoftDevice Controller are running on the same core.
+
+Building and running
+********************
+.. |sample path| replace:: :file:`samples/bluetooth/connection_event_trigger`
+
+.. include:: /includes/build_and_run.txt
+
+Testing
+=======
+
+After programming the sample to both development kits, test it by performing the following steps:
+
+1. Connect to both kits with a terminal emulator (for example, nRF Connect Serial Terminal).
+   See :ref:`test_and_optimize` for the required settings and steps.
+#. Reset both kits.
+
+   - You will be prompted::
+
+      Choose device role - type c (central) or p (peripheral):
+
+#. In one of the terminal emulators, type "c" to start the application on the connected board in the central (tester) role.
+#. In the other terminal emulator, type "p" to start the application in the peripheral (peer) role.
+#. Observe that the kits establish a connection.
+
+   - Each kit outputs the following information::
+
+       Press any key to switch to a 10ms connection interval and set up connection event trigger
+
+#. Press a key in either terminal.
+#. Observe that the link switches from a 100ms connection interval to a 10ms connection interval.
+#. The event trigger is configured to generate a software interrupt at each connection event for the link. During these software interrupts, a timestamp is recorded.
+#. Observe that these timestamps are printed in the terminal. Note that the delta between triggers generally matches the connection interval, but that if the controller is not able to schedule a radio event for a given point in time, the event trigger will also be dropped.
+#. Observe the central switches back to a 100ms connection interval.
+
+
+Sample output
+=============
+
+- The s/test/sample should output something similar to the following::
```

Comment:
When I said `s/test/sample` it meant [replace](https://www.gnu.org/software/sed/manual/html_node/The-_0022s_0022-Command.html) `test` by `sample`. Sorry should've been more explicit.